public/api/docs/v1/swagger.yaml

```diff
@@ -2277,6 +2277,24 @@ components:
               nullable: false
               format: date-time
               example: '2021-05-31T02:22:32.000Z'
+    ParticipantDeclarationResponse:
+      description: A single participant declaration.
+      type: object
+      required:
+      - data
+      properties:
+        data:
+          "$ref": "#/components/schemas/ParticipantDeclaration"
+    ParticipantDeclarationsResponse:
+      description: A list of participant declarations.
+      type: object
+      required:
+      - data
+      properties:
+        data:
+          type: array
+          items:
+            "$ref": "#/components/schemas/ParticipantDeclaration"
```

Comment:
these sections in v1 and v2 just moved, because of the changes to `spec/swagger_helper.rb`